### DISCUSSION TITLE
Erro send msg to page facebook

### DISCUSSION THREAD

**Discussion by AppDesv** [LiveHelperChat/livehelperchat#2059] [General]
Hello Remdex,

I'm using the fbmessenger extension, and I'm able to receive messages from the Facebook page in the chat, but I'm unable to respond to them.

When I reply to a message, nothing happens!

What could I have done wrong?

**Comment by remdex** [comment]
That one you will have to debug by digging around the extension itself :)

If you are talking about facebook messenger. That's the only function you need to debug,
https://github.com/LiveHelperChat/fbmessenger/blob/master/bootstrap/bootstrap.php#L756
